website/content/en/docs/olm-integration/generation.md

```diff
@@ -47,7 +47,25 @@ These values will persist when generating a bundle, so make necessary metadata c
 **For Go Operators only:** the command parses [CSV markers][csv-markers] from Go API type definitions, located
 in `./api` for single group projects and `./apis` for multigroup projects, to populate certain CSV fields.
 You can set an alternative path to the API types root directory with `--apis-dir`. These markers are not available
-to Ansible or Helm project types.
+to Ansible or Helm project types. Also note that the command will attempt to process local types defined in your API.
+If you import a package and use a type with the same name as a local type as a field in the local type it will loop infinitely.
+For example, the following local type definition will cause an infinite loop:
+```go
+type PodStatus struct {
+  SomeField string
+  // imported type with the same name will infinitely trigger
+  // the parser to process the local PodStatus type
+  Status v1.PodStatus 
+}
+```
+To prevent this, name the local type something different than the imported type:
+```go
+type PodStatusWrapper struct {
+  SomeField string
+  Status v1.PodStatus 
+}
+```
+Since the local type has a different name than the imported type the parser does not get stuck.
```

Comment:
Minimalism suggestion: I think you can cut this last sentence. If there is relevant information, I recommend bringing it up to line 61.

Reply:
Now that you point it out, I agree. Updated in commit https://github.com/operator-framework/operator-sdk/pull/6053/commits/ee8dd7f3f011f338fa8dcb228c9a532558c4273d